VersionControlUI/src/nodes/DiffComparisonPair.h

```diff
@@ -61,6 +68,9 @@ class VERSIONCONTROLUI_API DiffComparisonPair : public Super<Visualization::UINo
 		bool twoObjectPathsDefined();
 		QString comparisonName();
 
+		QList<ObjectPathCrumbData*> objectPathCrumbsDataOldNode();
```

Comment:
You are storing pointers here and you did not explicitly delete those in the destructor so there is a memory leak.

Unless you have a very specific reason why you need pointers (I'd like hear it if so), store directly the structures here and avoid the hassle of dealing with memory.